src/fsharp/FSharp.Core/map.fs

```diff
@@ -670,7 +670,8 @@ type Map<[<EqualityConditionalOn>]'Key, [<EqualityConditionalOn; ComparisonCondi
                 let m1 = e1.MoveNext() 
                 let m2 = e2.MoveNext()
                 (m1 = m2) && (not m1 || 
-                                 (let e1c, e2c = e1.Current, e2.Current
+                                 (let e1c = e1.Current
```

Comment:
Is this is a formatting change only? Both versions will be exactly the same. No tuple will be actually be created.

Reply:
yes I think it reduces confusion here.

Reply:
Ok, that's fine.